examples/guestbook/README.md

```diff
@@ -18,7 +18,7 @@ Note: This redis-master is *not* highly available.  Making it highly available w
 
 Use (or just create) the file `examples/guestbook/redis-master-controller.json` which describes a single [pod](../../docs/pods.md) running a redis key-value server in a container:
 
-Note that, although the redis server runs just with a single replica, we use [replication controller](../../docs/replication-controller.md) to enforce that exactly one pod keeps running (e.g. in a event of node going down, the replication controller will ensure that the redis master gets restarted on a healthy node).   This could result in data loss.
+Note that, although the redis server runs just with a single replica, we use [replication controller](../../docs/replication-controller.md) to enforce that exactly one [pod](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/pods.md) keeps running (e.g. in a event of node going down, the replication controller will ensure that the redis master gets restarted on a healthy node).   This could result in data loss.
```

Comment:
Sorry I know I didnt list it in the bulleted list in my last commnet but we should not linkify a term if an instance of the "term x" or "k8s + term x" already exists (we only want once instance per topic). For example, line 19 already has "single [pod]..." linkified so we should not include a second instance (-> line 21 this instance was linkified "exactly one [pod]..., making it the second instance of linkified "pods" in the document).